Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -53,6 +53,8 @@ const OVERRIDE_PROPS = [
   'accessibilityRole',
   'accessibilityStates',
   'accessibilityState',
+  'accessibilityActions',
```

Comment:
Looking at how TouchableWithoutFeedback was implemented, I think you might not have the limitation of whether the child of the TouchableWithoutFeedback supports accessibility actions by adding "accessibilityActions" and "onAccessibilityAction" to its Props, which are defined starting from line 66. (Additional changes are required. Please see how accessibilityState is added.)Could you try out this approach?

Reply:
It does not fix it. But for code consistency I added it.